packages/i18n/src/messages/localizedConstants.tsx

```diff
@@ -1576,6 +1576,11 @@ export const publishingGroups = defineMessages({
     id: "Mixlw/",
     description: "The publishing group called Executive Jobs",
   },
+  [PublishingGroup.Iap]: {
+    defaultMessage: "IAP",
+    id: "7ObbNV",
+    description: "The publishing group called IAP",
```

Comment:
Not sure if this gives the translators any helpful context for this acronym.  :laughing:

Reply:
Updated in [improve context of publishing group description](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6521/commits/fa251abc4e0f1d69e71a284fea0417bdd07f1d72)